### PR TITLE
Subatomic refactor

### PR DESCRIPTION
**Pull request by noroadsleft**
- refactored to use a matrix macro
  - I'm not crazy about the layout mock-ups in `subatomic.h`. Might take them out.
  - also guessed at the matrices for MIT and DIT layout, based on how @jackhumbert usually does his PCBs
- added a readme file
  - qmk.fm link leads to a page that's basically error 404
- added `info.json`

### CODE REVIEW

**Comment by drashna** [comment]
Well, Jack has different names in the planck rev6 branch.  It may be best to use those: 
https://github.com/qmk/qmk_firmware/blob/planck_rev6/keyboards/planck/planck.h

Namely 'LAYOUT_planck_1x2uC' and `LAYOUT_planck_2x2u`.

That way, when it's merged, we're using the same naming scheme.

**Comment by drashna** [comment]
Like the jj40, I know that you talked with jack about the olkb stuff.  Did you want to update this PR?

**Comment by noroadsleft** [comment]
Jack wants to wait. His exact words on Discord were:

> i'll want to wait until i have the design finalised - i don't want prototype stuff in qmk:master

Whether that means to just leave this dormant as-is, or to close it un-merged, I am not sure. I'm fine with either though.

**Comment by drashna** [comment]
I'm going to add the "On hold" label for now, and then once the planck rev6 stuff is merged, we'll see what to do from there.

**Comment by drashna** [comment]
@noroadsleft ?

Do we use `LAYOUT_ortho_5x15` or ......

**Comment by drashna** [comment]
@jackhumbert was this good to merge?

**Comment by drashna** [comment]
jaaaaaaak

**Comment by drashna** [comment]
@noroadsleft for the sake of sanity, and/or getting this closed, would it be possible to rename some of these layouts to match the planck rev6 type layouts?   Eg, `ortho_5x14x2x2U`

**Comment by noroadsleft** [comment]
Rebased and updated. Out of town this weekend. Any further changes will likely need to wait until after I get back home Sunday afternoon.

File: keyboards/subatomic/keymaps/default/keymap.c [hunk 1]
```diff
@@ -22,132 +24,131 @@ enum subatomic_keycodes {
...
+   * |   -  | Tab  |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  |   [  |   ]  |
+   * |------+------+------+------+------+------+-------------+------+------+------+------+------+------|
+   * |   =  | Esc  |   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   ;  |   "  |Enter |
+   * |------+------+------+------+------+------+------|------+------+------+------+------+------+------|
+   * | Pg Up| Shift|   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   /  | Home | End  |
+   * |------+------+------+------+------+------+------+------+------+------+------+------+------+------|
+   * | Pg Dn| Brite| Ctrl | Alt  | GUI  |Lower |    Space    |Raise | Left | Down |  Up  | Right|   \  |
+   * `-------------------------------------------------------------------------------------------------'
+   */
+  [_QWERTY] = LAYOUT_subatomic_mit(
```
Comment (drashna) [inline keyboards/subatomic/keymaps/default/keymap.c]:
```suggestion
  [_QWERTY] = LAYOUT_ortho_5x14_1x2uC(
```

File: keyboards/subatomic/keymaps/default/keymap.c [hunk 2]
```diff
@@ -22,132 +24,131 @@ enum subatomic_keycodes {
...
+   * |   -  | Tab  |   Q  |   W  |   F  |   P  |   G  |   J  |   L  |   U  |   Y  |   ;  |   [  |   ]  |
+   * |------+------+------+------+------+------+-------------+------+------+------+------+------+------|
+   * |   =  | Esc  |   A  |   R  |   S  |   T  |   D  |   H  |   N  |   E  |   I  |   O  |   "  |Enter |
+   * |------+------+------+------+------+------+------|------+------+------+------+------+------+------|
+   * | Pg Up| Shift|   Z  |   X  |   C  |   V  |   B  |   K  |   M  |   ,  |   .  |   /  | Home | End  |
+   * |------+------+------+------+------+------+------+------+------+------+------+------+------+------|
+   * | Pg Dn| Brite| Ctrl | Alt  | GUI  |Lower |    Space    |Raise | Left | Down |  Up  | Right|   \  |
+   * `-------------------------------------------------------------------------------------------------'
+   */
+  [_COLEMAK] = LAYOUT_subatomic_mit(
```
Comment (drashna) [inline keyboards/subatomic/keymaps/default/keymap.c]:
```suggestion
  [_COLEMAK] = LAYOUT_ortho_5x14_1x2uC(
```

File: keyboards/subatomic/keymaps/default/keymap.c [hunk 3]
```diff
@@ -22,132 +24,131 @@ enum subatomic_keycodes {
...
+   * |   [  | Tab  |   "  |   ,  |   .  |   P  |   Y  |   F  |   G  |   C  |   R  |   L  |   /  |   =  |
+   * |------+------+------+------+------+------+-------------+------+------+------+------+------+------|
+   * |   ]  | Esc  |   A  |   O  |   E  |   U  |   I  |   D  |   H  |   T  |   N  |   S  |   -  |Enter |
+   * |------+------+------+------+------+------+------|------+------+------+------+------+------+------|
+   * | Pg Up| Shift|   ;  |   Q  |   J  |   K  |   X  |   B  |   M  |   W  |   V  |   Z  | Home | End  |
+   * |------+------+------+------+------+------+------+------+------+------+------+------+------+------|
+   * | Pg Dn| Brite| Ctrl | Alt  | GUI  |Lower |    Space    |Raise | Left | Down |  Up  | Right|   \  |
+   * `-------------------------------------------------------------------------------------------------'
+   */
+  [_DVORAK] = LAYOUT_subatomic_mit(
```
Comment (drashna) [inline keyboards/subatomic/keymaps/default/keymap.c]:
```suggestion
  [_DVORAK] = LAYOUT_ortho_5x14_1x2uC(
```

File: keyboards/subatomic/keymaps/default/keymap.c [hunk 4]
```diff
@@ -22,132 +24,131 @@ enum subatomic_keycodes {
...
+   * |   _  | Tab  |   !  |   @  |   #  |   $  |   %  |   ^  |   &  |   *  |   (  |   )  |   {  |   }  |
+   * |------+------+------+------+------+------+-------------+------+------+------+------+------+------|
+   * |   +  | Esc  |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |   _  |   +  |   {  |   }  |      |      |
+   * |------+------+------+------+------+------+------|------+------+------+------+------+------+------|
+   * |      | Shift|  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |ISO ~ |ISO | |      |      |      |      |
+   * |------+------+------+------+------+------+------+------+------+------+------+------+------+------|
+   * |      |      |      |      |      |      |             |      | Next | Vol- | Vol+ | Play |      |
+   * `-------------------------------------------------------------------------------------------------'
+   */
+  [_LOWER] = LAYOUT_subatomic_mit(
```
Comment (drashna) [inline keyboards/subatomic/keymaps/default/keymap.c]:
```suggestion
  [_LOWER] = LAYOUT_ortho_5x14_1x2uC(
```

File: keyboards/subatomic/keymaps/default/keymap.c [hunk 5]
```diff
@@ -22,132 +24,131 @@ enum subatomic_keycodes {
...
+   * |   _  | Tab  |   1  |   2  |   3  |   4  |   5  |   6  |   7  |   8  |   9  |   0  |   {  |   }  |
+   * |------+------+------+------+------+------+-------------+------+------+------+------+------+------|
+   * |   +  | Esc  |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |   -  |   =  |   [  |   ]  |      |      |
+   * |------+------+------+------+------+------+------|------+------+------+------+------+------+------|
+   * |      | Shift|  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |ISO # |ISO / |      |      |      |      |
+   * |------+------+------+------+------+------+------+------+------+------+------+------+------+------|
+   * |      |      |      |      |      |      |             |      | Next | Vol- | Vol+ |      | Play |
+   * `-------------------------------------------------------------------------------------------------'
+   */
+  [_RAISE] = LAYOUT_subatomic_mit(
```
Comment (drashna) [inline keyboards/subatomic/keymaps/default/keymap.c]:
```suggestion
  [_RAISE] = LAYOUT_ortho_5x14_1x2uC(
```

File: keyboards/subatomic/keymaps/default/keymap.c [hunk 6]
```diff
@@ -22,132 +24,131 @@ enum subatomic_keycodes {
...
+   * |      |      | Reset|      |      |      |      |      |      |      |      |      |      |      |
+   * |------+------+------+------+------+------+-------------+------+------+------+------+------+------|
+   * |      |      |      |      |Aud on|AudOff|AGnorm|AGswap|Qwerty|Colemk|Dvorak|      |      |      |
+   * |------+------+------+------+------+------+------|------+------+------+------+------+------+------|
+   * |      |      |Voice-|Voice+|Mus on|MusOff|MidiOn|MidOff|      |      |      |      |      |      |
+   * |------+------+------+------+------+------+------+------+------+------+------+------+------+------|
+   * |      |      |      |      |      |      |             |      |      |      |      |      |      |
+   * `-------------------------------------------------------------------------------------------------'
+   */
+  [_ADJUST] = LAYOUT_subatomic_mit(
```
Comment (drashna) [inline keyboards/subatomic/keymaps/default/keymap.c]:
```suggestion
  [_ADJUST] = LAYOUT_ortho_5x14_1x2uC(
```

**Comment by drashna** [comment]
Awesome, looks good, and up to date!

@jackhumbert want to check off on this first?

**Pull request merged**